yarn.lock

```diff
@@ -3094,6 +3094,11 @@ lodash@^4.17.15, lodash@^4.17.21:
   resolved "https://registry.npmjs.org/lodash/-/lodash-4.17.21.tgz"
   integrity sha512-v2kDEe57lecTulaDIuNTPy3Ry4gLGJ6Z1O3vE1krgXZNrsQ+LFTGHVxVjcXPs17LhbZVGedAJv8XZ1tvj5FvSg==
 
+lru-cache@^10.0.1:
```

Comment:
A package by `isaacs`, that dude is a node legend.

Note this package requires Node version 16.14.0 or higher, which changes our minimum supported version up from 12. It's possible this installs and folks could use it no cache. So I'd use nvm and see if you can npm install this with node 12 or 16 is needed.

Reply:
Oh interesting - so I do see that we publish with `node:12` but we do not have an `engines` section in our `package.json`

Based on this schedule we should move up to 16 (https://stackoverflow.com/a/29349773/813154)

https://github.com/nodejs/release#release-schedule

Funny enough I just tried to install node 12 locally and it fails to compile - I think it predates the M1??? 😄 

--

The last version of `lru-cache` specifically supporting 12 is `7.6.1` - I just tried importing it into this branch and it is causing a boatload of trouble:

1. Has no default import so `import { LRUCache } from 'lru-cache'` does not work
2. We can enable `esModuleInterop` in tsconfig but this causes everything else to break

I'm feeling like I need to move forward with the latest version.

Reply:
Thanks for checking! Yeah sounds like node 12 is no good and we can defensibly change our minimum supported version to 16, so I agree we just do that.